### PR TITLE
Inverse Velocity Score

### PR DESCRIPTION
**Pull request by samuelshaner**
This PR adds a flux-weighted inverse velocity score that will be useful in kinetics calculations. While it might seem more natural to have a flux-weighted *velocity* score, the flux-weighted *inverse velocity* is required in kinetics calculations due to the 1/v term in the time-dependent neutron transport equation. The score is denoted `inverse-velocity`.

This PR builds on PR #368 and should not be reviewed until that PR is merged.

### CODE REVIEW

File: src/tally.F90 [hunk 1]
```diff
@@ -120,6 +127,27 @@ subroutine score_general(p, t, start_index, filter_index, i_nuclide, &
...
+          ! to count 'events' exactly for the inverse velocity
+          if (survival_biasing) then
+            ! We need to account for the fact that some weight was already
+            ! absorbed
+            score = p % last_wgt + p % absorb_wgt
+          else
+            score = p % last_wgt
+          end if
+          score = score / material_xs % total &
+               / (sqrt(2 * p % E / (MASS_NEUTRON * AMU_MEV)) * C_LIGHT)
```
Comment (paulromano) [inline src/tally.F90]: Use `TWO` instead of 2

File: src/tally.F90 [hunk 2]
```diff
@@ -120,6 +127,27 @@ subroutine score_general(p, t, start_index, filter_index, i_nuclide, &
...
+
+        else
+          ! For inverse velocity, we need no cross section
+          score = flux / (sqrt(2 * p % E / (MASS_NEUTRON * AMU_MEV)) * C_LIGHT)
```
Comment (paulromano) [inline src/tally.F90]: Use `TWO` instead of 2

File: src/tally.F90 (hunk 1 above)
Comment (paulromano) [inline src/tally.F90]: Also, CODATA provides neutron mass energy equivalent (939.565379 MeV/c^2), so you may want to consider having a constant `MASS_NEUTRON_MEV` instead.

File: docs/source/usersguide/input.rst [hunk 3]
```diff
@@ -1465,6 +1478,11 @@ The ``<tally>`` element accepts the following sub-elements:
           specified. Furthermore, it may not be used in conjunction with any
           other score.
 
+    :inverse-velocity:
+      The flux-weighted inverse velocity where the velocity is in units of
+      meters per second.  Note: The ``analog`` estimator is actually identical
```
Comment (paulromano) [inline docs/source/usersguide/input.rst]: Use the sphinx `.. note::` directive here

**Comment by samuelshaner** [comment]
@paulromano I just addressed your comments

**Comment by paulromano** [comment]
This one should be good to go -- are you planning on making any more changes @samuelshaner?

**Comment by samuelshaner** [comment]
Nope. I just pushed two minor changes to comments in the code and think it is now good to merge.

**Pull request merged**